### PR TITLE
Bugs - temp action for normalizing particip score

### PR DESCRIPTION
**Pull request by jicata**

### CODE REVIEW

File: Open Judge System/Web/OJS.Web/Areas/Administration/Controllers/ParticipantsController.cs [hunk 1]
```diff
@@ -205,5 +206,40 @@ public JsonResult UpdateParticipantAnswer([DataSourceRequest]DataSourceRequest r
...
             return this.GridOperation(request, model);
         }
+
+        public ActionResult NormalizeParticipants()
+        {
+            var problems = this.Data.Problems.All().ToList();
+            
+            foreach (var problem in problems)
+            {
+                var data = new OjsData();
```
Comment (Innos) [inline Open Judge System/Web/OJS.Web/Areas/Administration/Controllers/ParticipantsController.cs]: Remove the creation of a new OjsData, it is already passed to the constructor of the controller (you used this.Data 5 lines above)

File: Open Judge System/Web/OJS.Web/Areas/Administration/Controllers/ParticipantsController.cs [hunk 2]
```diff
@@ -135,9 +136,9 @@ public ActionResult Destroy([DataSourceRequest]DataSourceRequest request, ViewMo
 
         public JsonResult Contests(string text)
         {
-           var contests = this.Data.Contests
-                .All()
-                .Select(ContestViewModel.ViewModel);
+            var contests = this.Data.Contests
```
Comment (YanaSlavcheva) [inline Open Judge System/Web/OJS.Web/Areas/Administration/Controllers/ParticipantsController.cs]: why the space?

File: Open Judge System/Web/OJS.Web/Areas/Administration/Controllers/ParticipantsController.cs [hunk 3]
```diff
@@ -135,9 +136,9 @@ public ActionResult Destroy([DataSourceRequest]DataSourceRequest request, ViewMo
...
+                 .All()
```
Comment (YanaSlavcheva) [inline Open Judge System/Web/OJS.Web/Areas/Administration/Controllers/ParticipantsController.cs]: why the space?

File: Open Judge System/Web/OJS.Web/Areas/Administration/Controllers/ParticipantsController.cs [hunk 4]
```diff
@@ -135,9 +136,9 @@ public ActionResult Destroy([DataSourceRequest]DataSourceRequest request, ViewMo
...
+                 .Select(ContestViewModel.ViewModel);
```
Comment (YanaSlavcheva) [inline Open Judge System/Web/OJS.Web/Areas/Administration/Controllers/ParticipantsController.cs]: why the space?

File: Open Judge System/Web/OJS.Web/Areas/Administration/Controllers/ParticipantsController.cs (hunk 1 above)
Comment (jicata) [inline Open Judge System/Web/OJS.Web/Areas/Administration/Controllers/ParticipantsController.cs]: Creating a new instance of OjsData (and consequently of the underlying DbContext) speeds up the execution of the queries tremendously (upward of **x2200** times faster) and ensures no System.OutOfMemory exception is thrown.
Same OjsData instance -> 10 affected rows for 20 minutes
New OjsData for every iteration -> 11k affected rows in about 5 minutes

**Pull request merged**